jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -162,14 +162,20 @@
 		 */
 		public static final int UNCONDITIONAL = 0x20;
 		/*[ENDIF] Sidecar19-SE*/
-		
-		static final int INTERNAL_PRIVILEGED = 0x40;
 
-		/*[IF Sidecar19-SE-OpenJ9]
+		static final int INTERNAL_PRIVILEGED = 0x80;
```

Comment:
What is affected by changing the value of `INTERNAL_PRIVILEGED`?

Reply:
It is used to set `Lookup.accessMode`. Based upon its current usage, nothing should be affected based by changing its value.

Reply:
Please confirm that comment applies to all jdk versions (8, 11, 15, next).

Reply:
Yes, it does apply to all jdk versions (8, 11, 15, next).

Reply:
Thanks.